adserver/tasks.py

```diff
@@ -211,3 +212,35 @@ def daily_update_keywords(day=None):
                 KeywordImpression.objects.filter(pk=impression.pk).update(
                     **{impression_type: F(impression_type) + values["total"]}
                 )
+
+
+@app.task()
+def daily_update_uplift(day=None):
```

Comment:
Just a note that this task is not in celerybeat so it will not run automatically.

Reply:
Fixed.